app/models/job_invocation.rb

```diff
@@ -75,6 +75,8 @@ class JobInvocation < ApplicationRecord
 
   default_scope -> { order('job_invocations.id DESC') }
 
+  scope :latest_jobs, -> { unscoped.joins(:tasks).order('foreman_tasks_tasks.started_at DESC').authorized(:view_job_invocations).limit(5) }
```

Comment:
Most likely you want `joins(:task)` instead of `joins(:tasks)`. The first looks only at the "main" task of each job, the latter looks at all tasks within each job.